bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -3,13 +3,32 @@
 load '../helpers/load'
 ARCH_FOR_KUBERLR=amd64
 
+local_setup_file() {
+    if semver_eq "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION"; then
+        printf "Cannot upgrade from %s to %s\n" \
+            "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION" |
+            fail
+    fi
+    # It is undefined whether RD_KUBERNETES_VERSION is greater or less than
+    # RD_KUBERNETES_ALT_VERSION (and it's expected to flip in CI); actually
+    # compare them so we can expect to wipe data on the downgrade.
+    if semver_gt "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION"; then
+        export RD_KUBERNETES_VERSION_LOW=$RD_KUBERNETES_ALT_VERSION
+        export RD_KUBERNETES_VERSION_HIGH=$RD_KUBERNETES_VERSION
```

Comment:
I don't think these variables need to be exported, but of course it doesn't hurt...

Reply:
Per [BATS docs](https://bats-core.readthedocs.io/en/stable/writing-tests.html#setup-and-teardown-pre-and-post-test-hooks):

> Variables that are exported in `setup_file` will be visible to all following functions (`setup`, the test itself, `teardown`, `teardown_file`).

So I think that means the documented way to make it visible is to export it (this says nothing about what actually works, just what is supposed to work).

Reply:
Oh, you are right, this is `setup_file`. I thought it was a per-test `setup` function.